README.md

```diff
@@ -0,0 +1,15 @@
+# esptool-legacy-flasher-stub
+
+This project is based on the original flasher stub of [esptool](https://github.com/espressif/esptool). It will be deprecated in the near future. It is not advised to base projects on it because it won't receive official support from Espressif Systems. The new and supported flasher stub will the [esp-flasher-stub](https://github.com/esp-rs/esp-flasher-stub) project.
+
+## Issue Tracker
+
+Please report issues for [esptool](https://github.com/espressif/esptool/issues) if you encounter any.
```

Comment:
suggestion: `Please report issues in the [esptool repository](https://github.com/espressif/esptool/issues) if ...`